@xen-orchestra/lite/src/locales/en.json

```diff
@@ -0,0 +1,35 @@
+{
+  "add": "Add",
+  "add-filter": "Add filter",
```

Comment:
Why not using `CamelCase`?

Reply:
The eternal _naming things_ question ^^

These keys will not be used as properties, but as string. And I though `{{ $t("add-filter") }}` would be better than `{{ $t("addFilter") }}`

I'm ok to use whatever the team prefer, just tell me :)